docs/source/configuration.rst

```diff
@@ -75,25 +62,18 @@ Specify Configuration
 YAML files
 ~~~~~~~~~~
 
-You can specify configuration values in YAML files like the following:
+You can specify configuration values in YAML files. For example, if you want to modify 
+some of the workers memory configuration default values in distributed:
 
 .. code-block:: yaml
 
-   array:
-      chunk-size: 128 MiB
-
    distributed:
-     logging:
-       distributed: info
-       bokeh: critical
-       tornado: critical
-
-     scheduler:
-       work-stealing: True
-       allowed-failures: 5
+         workers: 
+            memory:
+               spill     : 0.85  # default: 0.7
+               target    : 0.75  # default: 0.6
+               terminate : 0.98  # default: 0.95
```

Comment:
Let's keep the `array.chunk-size` portion here as it will be good to show multiple top-level configuration options. Additionally, let's keep a uniform indentation size for this YAML code (I'd suggest 2 spaces as that's what used elsewhere on this page). 

```diff
diff --git a/docs/source/configuration.rst b/docs/source/configuration.rst
index 2fb65f38..4f3059bd 100644
--- a/docs/source/configuration.rst
+++ b/docs/source/configuration.rst
@@ -67,12 +67,15 @@ some of the workers memory configuration default values in distributed:

 .. code-block:: yaml

+   array:
+     chunk-size: 128 MiB
+
    distributed:
-         workers:
-            memory:
-               spill     : 0.85  # default: 0.7
-               target    : 0.75  # default: 0.6
-               terminate : 0.98  # default: 0.95
+     workers:
+       memory:
+         spill     : 0.85  # default: 0.7
+         target    : 0.75  # default: 0.6
+         terminate : 0.98  # default: 0.95


 These files can live in any of the following locations:
```